File: RepoTransBench/target_projects/Java/C#/sockeqwe_fragmentargs/tests/original/NoneArgsBundlerTest.cs
using Xunit;

namespace ProjectName.Tests.Original.Bundler
{
    public class NoneArgsBundler
    {
        private static readonly NoneArgsBundler _instance = new NoneArgsBundler();

        public static NoneArgsBundler Get() => _instance;

        public object? Put(string key, object value, Bundle bundle) => null;

        public object? Get(string key, Bundle bundle) => null;
    }

    public class Bundle { }

    public class NoneArgsBundlerTest
    {
        [Fact]
        public void TestGetInstanceReturnsSingleton()
        {
            var a = NoneArgsBundler.Get();
            var b = NoneArgsBundler.Get();
            Assert.True(object.ReferenceEquals(a, b));
        }

        [Fact]
        public void TestPutReturnsNull()
        {
            Assert.Null(NoneArgsBundler.Get().Put("key", 123, new Bundle()));
        }

        [Fact]
        public void TestGetReturnsNull()
        {
            Assert.Null(NoneArgsBundler.Get().Get("key", new Bundle()));
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/sockeqwe_fragmentargs/tests/original/noneargsbundler_test.go
package original

import (
	"testing"
)

type NoneArgsBundler struct{}

var noneArgsBundlerSingleton = &NoneArgsBundler{}

func NoneArgsBundlerGet() *NoneArgsBundler {
	return noneArgsBundlerSingleton
}

func (*NoneArgsBundler) Put(key string, value interface{}, bundle map[string]interface{}) interface{} {
	return nil
}

func (*NoneArgsBundler) Get(key string, bundle map[string]interface{}) interface{} {
	return nil
}

func TestGetInstanceReturnsSingleton(t *testing.T) {
	a := NoneArgsBundlerGet()
	b := NoneArgsBundlerGet()
	if a != b {
		t.Errorf("get() should always return same instance")
	}
}

func TestPutReturnsNull(t *testing.T) {
	got := NoneArgsBundlerGet().Put("key", 123, map[string]interface{}{})
	if got != nil {
		t.Errorf("put should always return nil, got %v", got)
	}
}

func TestGetReturnsNull(t *testing.T) {
	got := NoneArgsBundlerGet().Get("key", map[string]interface{}{})
	if got != nil {
		t.Errorf("get should always return nil, got %v", got)
	}
}